### PR TITLE
Update ide.md

### PR DESCRIPTION
**Pull request by alexandercyu**
Add introduction for IDE section

### CODE REVIEW

File: tutorials/users/ide/ide.md [hunk 1]
```diff
@@ -1 +1,9 @@
 # IDE
```
Comment (mishaschwartz) [inline tutorials/users/ide/ide.md:1]: Change this to "Interactive Development Environment"

(don't use acronyms before they've been introduced in text)

File: tutorials/users/ide/ide.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+Jupyter is a web-based development environment well suited for arranging workflows in
```
Comment (mishaschwartz) [inline tutorials/users/ide/ide.md:3]: I'd move the last sentence first... talk about Marble in the first sentence, introduce the concept of an IDE, and _then_ discuss jupyter

Comment (mishaschwartz) [inline tutorials/users/ide/ide.md:3]: Let's talk about JupyterLab specifically (not just Jupyter)

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

File: tutorials/users/ide/ide.md (hunk 1 above)
Comment (alexandercyu) [reply tutorials/users/ide/ide.md:1]: Changed to use full wording.

File: tutorials/users/ide/ide.md (hunk 2 above)
Comment (alexandercyu) [reply tutorials/users/ide/ide.md:3]: Changed to keep the topic about JupyterLab

**Review by mishaschwartz — APPROVED** [review]

File: tutorials/users/ide/ide.md (hunk 2 above)
Comment (alexandercyu) [reply tutorials/users/ide/ide.md]: Did another write-up.  Not sure if I should keep the part in the IDE explanation  about "source code editor,
build automation tools, and a debugger."

**Pull request merged**